pandas-stubs/core/frame.pyi

```diff
@@ -38,6 +38,7 @@ from pandas.core.window.rolling import (
     Rolling,
     Window,
 )
+import xarray as xr
```

Comment:
`xarray` is an optional dependency for pandas.  Concerned that this will cause things to break for people who don't have `xarray` installed.

Reply:
I think it should be fine: if it isn't installed, mypy/pyright will resolve it as Any/Unknown